charts/INSTALLATION.md

```diff
@@ -0,0 +1,279 @@
+Instrukcja instalacji będzie po polsku (może przydać się przy dokumentacji).
+
+# 1. Skrócona instrukcja
+
+```shell
+cd charts/
+helm install access-manager kam
+```
+Tyle wystarczy by zainstalować helm charta na klastrze. Więcej wyjaśnień poniżej.
+
+# 2. Komendy helm
+
+```shell
+helm install [name] [chart name] --namespace <namespace>
+```
+Komenda instalująca helm chart na klastrze.
+`[name]` odnosi się do tego jak ta konkretna instancja helm chartu będzie nazywana. Wpływa na nazwy zasobów.
+`[chart name]` nazwa helm charta (folderu.) W naszym przypadku `kam` z folderu `charts`.
+Flaga `--namespace <namespace>` jest opcjonalna i pozwala zainstalować chart w wybranym namespace.
+
+[Więcej informacji](https://helm.sh/docs/helm/helm_install/)
+
+---
+
+```shell
+helm uninstall <name>
+```
+Komenda pozwalająca odinstalować release charta. Po wykonaniu wszystkie zasoby związane z daną instancją charta zostaną usunięte.
+
+[Więcej informacji](https://helm.sh/docs/helm/helm_uninstall/)
+
+---
+
+```shell
+helm upgrade <release> <chart>
+```
+Komenda uaktualniająca wybrany release (instancję charta z określoną nazwą). Wszystkie zmiany wprowadzone w charcie zostaną zastosowane na klastrze. Numer revision zwiększy się o jeden.
+
+[Więcej informacji](https://helm.sh/docs/helm/helm_upgrade/)
+
+---
+
+```shell
+helm rollback <release> <revision>
+```
+Komenda cofająca release do podanej rewizji. Jeżeli revision będzie puste lub równe 0 release zostanie cofnięty do poprzedniej wersji. Rollback tworzy kolejną rewizję (zwiększa revision).
+
+[Więcej informacji](https://helm.sh/docs/helm/helm_rollback/)
+
+---
+
+```shell
+helm template [chart name]
+```
+Komenda renderuje lokalnie szablony i wyświetla jak będą wyglądały. Wartości które normalnie zostałby uzyskane z klastra będą dodane sztucznie. Nie będzie też serwerowych testów (np. czy na klastrze istnieje już taki release).
+
+[Więcej informacji](https://helm.sh/docs/helm/helm_template/)
+
+---
+
+```shell
+helm install [name] [chart name] --dry-run --debug
+```
+Komenda działająca podobnie do `helm template`, z tym że sprawdzi również czy nie ma konfliktujących zasobów na klastrze.
+[Więcej infromacji](https://helm.sh/docs/chart_template_guide/debugging/)
+
+---
+
+```shell
+helm create <name>
+```
+Komenda tworząca nowy chart folder wraz z często używanymi plikami.
```

Comment:
Na moje to nie powinno się znaleźć w dokumentacji - jeżeli ktoś będzie chciał się tego dowiedzieć, to napisze sobie helm --help, ewentualnie wpisze w googla. Ale może niech @Nightbaka się jeszcze wypowie

Reply:
Może i jest to w --help, ale nie wiem jak ty, ale jak niespecjalnie lubię to czytać w tamtej formie.

Reply:
Przydało mi się to przeczytać raz może Verze też się przyda jak pierwszy raz będzie se to odpalać